src/components/modules/banners/ausMoment/components/AusBannerCloseButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import { ThemeProvider } from '@emotion/react';
+import { Button, buttonBrand } from '@guardian/src-button';
+import { SvgCross } from '@guardian/src-icons';
+import { neutral } from '@guardian/src-foundations/palette';
+import { brand } from '@guardian/src-foundations';
+
+const closeButtonStyles = css`
```

Comment:
the close button should be light blue (#04ffff)

Reply:
I think on Figma it says #052962 (brand.400)

Reply:
👍 my misunderstanding!